src/util/prometheus.js

```diff
@@ -797,88 +550,6 @@ class Prometheus {
         type: 'histogram',
         labelNames: ['destination', 'version'],
       },
-      {
-        name: 'creation_time',
-        help: 'creation_time',
-        type: 'histogram',
-        labelNames: ['transformerVersionId', 'language', 'identifier', 'publish', 'testMode'],
-      },
-      { name: 'get_tracking_plan', help: 'get_tracking_plan', type: 'histogram', labelNames: [] },
-      { name: 'createivm_duration', help: 'createivm_duration', type: 'histogram', labelNames: [] },
-      {
-        name: 'fetchV2_call_duration',
```

Comment:
Why are we removing these? Shouldn't we update the labels?

Reply:
these were moved to end of the list